.circleci/config.yml

```diff
@@ -11,111 +19,136 @@ commands:
-      - notify-buildcop
-
+    - run:
+        command: |
+          apt-get update
+          apt-get install -y git
+          apt-get install -y openssh-server
+    - checkout
+    - if-php-modified:
+        steps:
+        - run:
+            name: PHP 5.6 Magento 1.9
+            command: |
+              mkdir ./artifacts
+              export TEST_ENV=php56-mage19
+              PHPUNIT_PHAR=tests/unit/phpunit-5.7.9.phar MAGENTO_VERSION=magento-mirror-1.9.3.6 tests/scripts/ci.sh
+        - store_artifacts:
+            path: ./artifacts
+        - notify-buildcop
   integration-sandbox:
     docker:
-      - image: circleci/openjdk:latest-node
-      - image: selenium/standalone-chrome
+    - image: circleci/openjdk:latest-node
+    - image: selenium/standalone-chrome
     steps:
-      - checkout
-      - run:
-          name: Integration tests (only for circleci branch)
-          command: |
-              echo "Running integration tests"
-              git clone git@github.com:BoltApp/integration-tests.git
-              cd integration-tests
-              npm install
-              mkdir -p ./screenshots
-              JUNIT_REPORT_DIR=./test-results SCREENSHOT_DIR=./screenshots TEST_ENV=sandbox npm run test-checkout-magento1
-      - store_test_results:
-          path: ./integration-tests/test-results
-      - store_artifacts:
-          path: ./integration-tests/screenshots
-      - run:
-          name: Generate Report
-          command: |
-            cd integration-tests
-            npm run generate-report
-          when: always
-      - store_artifacts:
-          path: ./integration-tests/allure-report
-      - notify-buildcop
-
+    - checkout
+    - run:
+        name: Integration tests (only for circleci branch)
+        command: |
+          echo "Running integration tests"
+          git clone git@github.com:BoltApp/integration-tests.git
+          cd integration-tests
+          npm install
+          mkdir -p ./screenshots
+          JUNIT_REPORT_DIR=./test-results SCREENSHOT_DIR=./screenshots TEST_ENV=sandbox npm run test-checkout-magento1
+    - store_test_results:
+        path: ./integration-tests/test-results
+    - store_artifacts:
+        path: ./integration-tests/screenshots
+    - run:
+        name: Generate Report
+        command: |
+          cd integration-tests
+          npm run generate-report
+        when: always
+    - store_artifacts:
+        path: ./integration-tests/allure-report
+    - notify-buildcop
+orbs:
+  slack: circleci/slack@3.3.0
+  swissknife: roopakv/swissknife@0.25.0
+version: 2.1
 workflows:
-  version: 2
   tests:
     jobs:
-      - integration-sandbox:
-          filters:
-            branches:
-              only:
-                - circleci
-      - unit-php55-mage18
-      - unit-php55-mage19
-      - unit-php56-mage18
-      - unit-php56-mage19
+    - integration-sandbox:
+        filters:
+          branches:
+            only:
+            - circleci
```

Comment:
who actually runs this? if no one, either we need to either fix it, or nuke it.

Reply:
Nuked